internal/service/account/resource_account_authentication_test.go

```diff
@@ -35,6 +46,109 @@ func TestAccAivenAccountAuthentication_basic(t *testing.T) {
 	})
 }
 
+func testAccAccountAuthenticationResourceSAMLCertificate(rName, certificate string) string {
```

Comment:
It seems to be that we can fit all the newly created tests into one test function with multiple steps, for example: 
`TestAccAivenAccountAuthentication_saml_valid_certificate` is basically the same with `TestAccAivenAccountAuthentication_saml_valid_certificate_update_delete` but we just have one extra `update` step.  

Moreover we no longer need the `TestAccAivenAccountAuthentication_basic` test so please drop it or move all the new tests inside.

Reply:
basic test doesn't have certificate file